components/gitpod-protocol/src/protocol.ts

```diff
@@ -123,25 +123,6 @@ export namespace User {
         return undefined;
     }
 
-    export function hasPreferredIde(user: User) {
```

Comment:
Should getPrimaryEmail, getName deleted as well above?

Reply:
yes! consider removed with next update